.github/workflows/build_package.yml

```diff
@@ -0,0 +1,98 @@
+name: Build Package
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build1:
+    runs-on: ubuntu-latest
```

Comment:
give some appropriate names for the _workflow jobs_ instead of just `build1` and `build2`.